data_prep/data_utils.py

```diff
@@ -0,0 +1,45 @@
+from data_prep.agnews_data import *
+from data_prep.graph_dataset import *
+from data_prep.imdb_data import *
+from data_prep.pure_graph_dataset import *
+from data_prep.reuters_data import *
+from data_prep.roberta_graph_dataset import *
+from data_prep.roberta_dataset import *
+
+
+def get_dataloaders(model, b_size, data_name):
+    corpus = get_data(data_name)
+    additional_params = {}
+
+    if model == 'roberta':
+        train_dataloader = RobertaDataset(corpus.train).as_dataloader(b_size, shuffle=True)
+        test_dataloader = RobertaDataset(corpus.test).as_dataloader(b_size)
+        val_dataloader = RobertaDataset(corpus.val).as_dataloader(b_size)
+
+        additional_params['num_classes'] = corpus.num_classes
+
+        return train_dataloader, test_dataloader, val_dataloader, additional_params
```

Comment:
This return here is not necessary right? You can return everything in line 34 (and unindent that line)

Reply:
The order is different there. I assume that's not a big deal tho